buddysuite/SeqBuddy.py

```diff
@@ -4827,11 +4887,11 @@ def duck_type(_arg, **kwargs):  # Feed feature_attrs into kwargs
         if not in_args.out_format:
             seqbuddy.out_format = "gb"
 
-        _enzymes, order, min_cuts, max_cuts = _prepare_restriction_sites(in_args.find_restriction_sites[0])
+        _enzymes, order, min_cuts, max_cuts, topology = _prepare_restriction_sites(in_args.find_restriction_sites[0])
 
         clean_seq(seqbuddy)
         try:
-            find_restriction_sites(seqbuddy, tuple(_enzymes), min_cuts, max_cuts, quiet=in_args.quiet)
+            find_restriction_sites(seqbuddy, tuple(_enzymes), min_cuts, max_cuts, topology, quiet=in_args.quiet)
         except TypeError as e:
             _raise_error(e, "find_restriction_sites")
```

Comment:
Let's add the third argument:
`check_string='Unable to identify restriction'`

Also modify the _raise_error() called in in_silico_digest